packages/-ember-data/index.js

```diff
@@ -9,6 +9,8 @@ module.exports = Object.assign({}, addonBaseConfig, {
   shouldRollupPrivate: true,
   externalDependenciesForPrivateModule() {
     return [
+      'ember',
```

Comment:
Why did we have to add this?

Reply:
<img width="851" alt="Screen Shot 2021-03-26 at 3 28 54 PM" src="https://user-images.githubusercontent.com/7374640/112689231-ffc73680-8e47-11eb-8550-28502fc8a78b.png">